data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2403 @@
+			choice
+				`	(Stay here and get ready for your next shot.)`
+					goto ready
+				`	(Go to the cover on the right.)`
+			`	You try to head for the cover but he is too fast, and you receive a shot on your leg, making you fall. It hurts a lot but nothing seems to be broken, and soon after you cannot feel anything in your leg at all. You opponent comes to you and helps you to get up. "Pain is a good teacher, sometimes," he says. Then he goes back to fight with his friends, seemingly in teams.`
+				goto heyi
+			
+			label ready
+			`	You try to shoot him but he is too fast and you too inexperienced. By moving still you made an easy target for him though, you fall as you receive a shot to your shoulder. It hurts a lot but nothing seems to be broken, and soon after you cannot feel anything in your leg at all. You opponent comes to you and helps you to get up. "Pain is a good teacher, sometimes," he says. Then he goes back to fight with his friends, seemingly in teams.`
+			
+			label heyi
+			`	It seems like your fight has attracted the attention of many of the Unfettered, including He Yi. He comes to speak to you, "That was a nice fight, especially considering this most be your first!"`
+			choice
+				`	"Can I help you with something?"`
+					goto speak
+				`	"Well, I lost."`
+					goto lost
+				`	"It was quite some fun, do you do this often?"`
+			`	"Fun that it is! Some of us decide to do it yes, we most be ready to repel a ground assault."`
+				goto speak
+			
+			label lost
+			`	"Were you expecting to win the fight? Those of us who choose to fight are well trained to do so, even if we do not overlook our studies whilst doing so. You may not have won the fight, but you have won something else."`
+			choice
+				`	"And what is that?"`
+				`	"Can I help you with anything?"`
+					goto speak
+			`	"Experience? Fun? There is always something to be gained when there is not the risk of losing your life."`
+				goto speak
+			
+			label challenge
+			`	You try to get a challenge with him, but you are not experienced enough for them to allow it. You did attract his attention however and he comes to speak to you.`
+				goto speak
+			label nope
+			`	You look at them battling for so long you do not see the time pass, then you realize you are being watched. It is He Yi, he looks satisfied of your interest in this fight. "You should try it someday," he says.`
+			
+			label speak
+			`	"As you have noticed, we do this training quite often. Because our rifles are not deadly on the first shots, we almost never get anyone killed. But it still happens too often, especially with some of the older ones who are more fragile... Or with the young ones who are too... enthusiast. Do you happen to know any weapons that would not have such weakness?"`
```

Comment:
```suggestion
			`	"As you have noticed, we do this training quite often. Because our rifles are not deadly on the first shots, we almost never have anyone get killed. But it still happens too often, especially with some of the older ones who are more fragile... Or with the young ones who are too, enthusiastic. Do you happen to know any weapons that would not have such weakness?"`
```